apps/qubit/modules/settings/actions/securityAction.class.php

```diff
@@ -62,11 +62,13 @@ protected function populateSecurityForm()
     $limitAdminIp = QubitSetting::getByName('limit_admin_ip');
     $requireSslAdmin = QubitSetting::getByName('require_ssl_admin');
     $requireStrongPasswords = QubitSetting::getByName('require_strong_passwords');
+    $googleMapsApiKey = QubitSetting::getByName('google_maps_api_key');
```

Comment:
Hrm... maybe we can put it into global until there are other API keys to group with it.